packages/cubicle/cubicle.1.2.0/opam

```diff
@@ -0,0 +1,29 @@
+opam-version: "2.0"
+maintainer: "alainmebsout@gmail.com"
+authors: [
+  "Sylvain Conchon"
+  "Alain Mebsout"
+]
+homepage: "http://cubicle.lri.fr"
+license: "Apache-2.0"
+bug-reports: "https://github.com/cubicle-model-checker/cubicle/issues"
+dev-repo: "git+https://github.com/cubicle-model-checker/cubicle.git"
+build: [
+  ["./configure" "--prefix" prefix]
+  [make]
+]
+depends: [
+  "ocaml" {>= "4.08.0"}
+  "ocamlfind"
+  "num"
+]
+depopts: ["functory"]
+conflicts: [
+  "functory" {< "0.5"}
+]
+synopsis: "SMT based model checker for parameterized systems"
+extra-files: ["cubicle.install" "md5=ba6d18615d00544948c96638b6c8d900"]
+url {
+  src: "https://github.com/cubicle-model-checker/cubicle/archive/refs/tags/1.2.0.tar.gz"
```

Comment:
I just realized that Cubicle uses this specific artifact for releases (which I uploaded just now) :
`https://github.com/cubicle-model-checker/cubicle/releases/download/1.2.0/cubicle-1.2.0.tar.gz`
That should make the CI happier (it includes the missing configure script).

Reply:
Done. Thanks a lot!